custom_components/powercalc/data/signify/LTA009/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "tuya",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 1,
+        "SLEEP_TIME": 2,
+        "VERSION": "v0.19.15:docker"
+    },
+    "name": "LTA009",
```

This has to be the full name of the light. Will be shown in supported models list.

```suggestion
    "name": "Hue White Ambiance 1100 E27 Bluetooth",
```